spec/models/manageiq/providers/autosde/storage_manager/refresher_v2_spec.rb

```diff
@@ -4,10 +4,10 @@
   let(:ems) do
     FactoryBot.create(:autosde_storage_manager,
                       :with_autosde_credentials,
-                      :hostname => Rails.application.secrets.autosde[:appliance_host])
+                      :hostname => credentials_autosde_host)
```

Comment:
I needed to change this and realized the test file's name was incorrect and was never run in CI or locally unless you specifically ran the path to the file.